hash/megadriv.xml

```diff
@@ -10054,6 +10111,32 @@ https://tcrf.net/World_Championship_Soccer_(Genesis)
 -->
 	</software>
 
+	<software name="zerotol_de" cloneof="zerotol">
+		<description>Zero Tolerance (Germany)</description>
+		<year>1994</year>
+		<publisher>Accolade</publisher>
+		<info name="developer" value="Technopop" />
+		<info name="language" value="English" />
+		<info name="serial" value="T-119146-00" />
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="2097152">
+				<rom name="Zero Tolerance (Germany) (En).bin " size="2097152" crc="6fb49ff8" sha1="c2ea98f595cb477495a9d326ff25d91f3dbfa90b" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="zerotol_swt" cloneof="zerotol">
+		<description>Zero Tolerance (from "QUByte Classics: Zero Tolerance Collection by PIKO" on Switch eShop)</description>
+		<year>2022</year>
+		<publisher>Piko Interactive / QUByte Interactive</publisher>
+		<info name="developer" value="Technopop" />
+		<info name="language" value="English" />
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="2097152">
+				<rom name="ZeroTolerance_Piko.bin " size="2097152" crc="de1f0797" sha1="f31737565359eb55d9c23954783bd0d94458c776" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Do we really want to be adding these images extracted from emulated re-releases?  This doesn’t have a corresponding physical release, does it?

Reply:
I think it might make sense to keep these sets (even though they don't have a physical release). It's still an alternate version of the original game. But if you think it makes sense to remove those entries, I will.

Reply:
Yeah, remove these please.

Reply:
Why would we not preserve digital releases for old systems? Many new ZX Spectrum games don't have a physical release, either.

Reply:
https://docs.mamedev.org/contributing/softlist.html#introduction

> It is not our intention to propagate corrupted, truncated, defaced, watermarked, or otherwise bad media images

Reply:
It's a new official release with all the needed licenses legally acquired.

Reply:
That doesn't mean we are interested in it, period.

Reply:
> Yeah, remove these please.

Done it.

Reply:
Zero Tolerance Underground is a new sequel with different levels and mechanics, I don't think it should be removed...